radis/io/hitran.py

```diff
@@ -284,6 +285,8 @@ def hit2df(
     # but files that have partly relevant lines are fully loaded.
     # Note : cache file is generated with the full line list.
 
+    df = reduce_mem_usage(df)
```

Comment:
note : this doesn't change the format in the cache HDF5 stored on disk (already written above).